org.eclipse.draw2d/META-INF/MANIFEST.MF

```diff
@@ -14,7 +14,8 @@ Export-Package: org.eclipse.draw2d,
  org.eclipse.draw2d.text,
  org.eclipse.draw2d.widgets,
  org.eclipse.draw2d.zoom
-Import-Package: com.ibm.icu.text
+Import-Package: com.ibm.icu.text,
```

Comment:
I think this import package can be removed now or?

Reply:
It's still used by the [BidiProcessor](https://github.com/eclipse/gef-classic/blob/master/org.eclipse.draw2d/src/org/eclipse/draw2d/text/BidiProcessor.java)

Reply:
Oh missed that.

Reply:
But I think I can use the same mechanism to get rid of that dependency as well. The only trick part being `Bidi.requiresBidi(...)` a static method call...